test/ResponseTest.php

```diff
@@ -152,12 +152,18 @@ public function ianaCodesReasonPhrasesProvider(): array
             $value       = $value;
             $description = $description;
```

Comment:
What this assignment achieves?

Reply:
Don't know. It's not part of the changes I introduced, but I'll remove them.